assets/css/index.css

```diff
@@ -162,6 +162,11 @@ hr {
   vertical-align: sub;
 }
 
+.navbar .sidebar-toggler{
```

Comment:
This works only for the toggler, please also add a case for `.close-sidebar` with it

Reply:
okay

Reply:
@shahednasser I have fixed the changes